src/xcode/ENA/ENA/Source/Scenes/Checkin/QRCodeScanner/CheckinQRCodeScannerViewController.swift

```diff
@@ -48,27 +47,127 @@ class CheckinQRCodeScannerViewController: UIViewController {
 
 	// MARK: - Private
 
+	private var focusView: QRScannerFocusView!
```

Comment:
```suggestion
	private let focusView = QRScannerFocusView()
```
why use a var here, it never gets changed